buyer-app/buyer-app/src/components/buyer/Buyer.js

```diff
@@ -184,6 +224,7 @@ const mapStateToProps = state => ({
   boughtDataOrders: DataOrdersByAddress.getBoughtDataOrders(state),
   closedDataOrders: DataOrdersByAddress.getClosedDataOrders(state),
   dataOrdersAddress: DataOrdersAddresses.getDataOrdersAddresses(state),
+  dataOrdersAddressAmount: DataOrdersAddressesAmount.getDataOrdersAddressesAmount(state),
```

Comment:
I think that the name `dataOrdersAddressesAmount` is not clear enough, but I couldn't come up with a better one 😜. We can review it in future PRs when we come back to this portion of the UI.